modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/persistence/Database.java

```diff
@@ -159,6 +160,41 @@ public boolean saveProperty(final Property property) {
     });
   }
 
+  /**
+   * Save a property to the database. This is either an insert or an update operation.
+   */
+  public boolean saveAssetProperty(final Property property) {
```

Comment:
Necessary? I didn't find where this method was called.

Reply:
Good catch, this was the old way and I missed removing this method.